android/src/main/java/com/inprogress/reactnativeyoutube/YouTubeView.java

```diff
@@ -123,7 +123,7 @@ public void playerViewDidBecomeReady() {
     public void didChangeToSeeking(int milliSeconds) {
         WritableMap event = Arguments.createMap();
         event.putString("state", "seeking");
-        event.putInt("currentTime", milliSeconds / 1000);
+        event.putDouble("currentTime", milliSeconds / 1000.0);
```

Comment:
No exists `putFloat`

Reply:
Checked. Thanks! Little interesting that the original author used sometime milliSecs and sometimes secs.